os/gl_window.h

```diff
@@ -15,8 +15,19 @@
 #define GL_UNSIGNED_SHORT_5_5_5_1 0x8034
 #endif
 
+#ifdef linux || _WIN32
+#include <GL/glew.h>
```

Comment:
I get build issues on Linux with the added <GL/glew.h> import.

Reply:
Fixed it too